lib/RoutingList/RoutingListView/RoutingListView.js

```diff
@@ -0,0 +1,190 @@
+import { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router';
+
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Button,
+  checkScope,
+  Col,
+  collapseAllSections,
+  ConfirmationModal,
+  ExpandAllButton,
+  expandAllSections,
+  HasCommand,
+  Icon,
+  KeyValue,
+  LoadingView,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+import { AppIcon, IfPermission } from '@folio/stripes/core';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+
+import {
+  useShowCallout,
+  useToggle,
+} from '../../hooks';
+import { handleKeyCommand } from '../../utils';
+import {
+  useGoBack,
+  useRoutingList,
+  useRoutingListMutation,
+} from '../hooks';
+import { RoutingListUsers } from '../RoutingListUsers';
+import { getAppName } from '../utils';
+
+export const RoutingListView = ({ routingListUrl }) => {
+  const accordionStatusRef = useRef();
+  const showCallout = useShowCallout();
+  const { id } = useParams();
+  const appName = getAppName();
+
+  const [isDeleteConfirmationVisible, toggleDeleteConfirmation] = useToggle(false);
+
+  const { routingList, isLoading } = useRoutingList(id);
+  const { deleteRoutingList } = useRoutingListMutation();
+  const onClose = useGoBack(routingList?.poLineId);
```

Comment:
=> `history.push('.../undefined')`? Please provide back path based on the place it used